spec/requests/api/v0/get_manifest_spec.rb

```diff
@@ -14,6 +14,17 @@
         expect(response).to match_response_schema("get-manifest")
       end
 
+      it 'finds by manifest tracking number' do
+        manifest = create(:manifest, activity_id: 1, document_id: 2)
+        manifest.reload
+
+        get "/api/v0/manifests/#{manifest.tracking_number}",
```

Comment:
If I was using this API, I would use the UUID, especially if iterating over search results, since the `id` is a top-level attribute.

IMO good API design follows Postel's law (https://en.wikipedia.org/wiki/Robustness_principle).